keras_nlp/layers/sine_position_encoding.py

```diff
@@ -31,6 +31,11 @@ class SinePositionEncoding(keras.layers.Layer):
     positional encoding the same size as the embedded token tensor, which
     can be added directly to the embedded token tensor.
 
+    This layer optionally accepts `tf.RaggedTensor`s as inputs to process
```

Comment:
Thanks! I think this makes sense as it will get our two position embedding options in sync.

We should add some tests though. Probably some you could mirror from the position embedding tests.

Reply:
Done. I've added the tests